examples/docker-build/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "docker-build",
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt-start",
+    "generate": "nuxt generate"
+  },
+  "dependencies": {
+    "cross-env": "latest",
+    "nuxt-start": "latest"
+  },
+  "devDependencies": {
+    "nodemon": "latest",
```

Comment:
Is `nodemon` required?

Reply:
removed `nodemon` as it is not required.